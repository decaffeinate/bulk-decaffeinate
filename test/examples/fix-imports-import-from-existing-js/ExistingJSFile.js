import ConvertedCoffeeFile from './ConvertedCoffeeFile';

console.log(ConvertedCoffeeFile.firstValue);
console.log(ConvertedCoffeeFile.secondValue);
