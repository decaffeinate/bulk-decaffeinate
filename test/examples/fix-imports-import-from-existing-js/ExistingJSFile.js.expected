import ConvertedCoffeeFile, * as ConvertedCoffeeFileExports from './ConvertedCoffeeFile';

console.log(ConvertedCoffeeFile.firstValue);
console.log(ConvertedCoffeeFileExports.secondValue);
